genai-on-vertex-ai/agents/reasoning_engine/building_and_deploying_agents_with_gemini.ipynb

```diff
@@ -0,0 +1,851 @@
+{
```

Comment:
<p>Can you provide link to files that user can upload that goes with the prompt?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>